distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -37,6 +37,7 @@ No configuration.
  start.solr=true
  solr.client=HttpSolrClient
  solr.http.port=8994
+ solr.max.heap.size=2g
```

Comment:
✏️ Unrelated to this change, but I just noticed that this file still says that SolrCloud is a Beta feature. We may want to do a quick scan of the documentation and remove any mention of it being Beta.

Reply:
Interesting. I thought it was still beta. I did a text search removed the (single) reference to Solr as a beta feature.